### DISCUSSION TITLE
See what tittle of the removed video

### DISCUSSION THREAD

**Discussion by niklas1212** [ReVanced/revanced-discussions#144] [Question] [answered]
Sometimes you want to re watch the video and then you come across "deleted videos" the problem with this is that you do not know which videos it was.

It would be great if Revanced could save the titles on the videos so you know what was removed.

**Comment by oSumAtrIX** [comment, marked as answer]
YouTube prevents access to the API. ReVanced will not be able to lift that restriction, it's a limitation, sorry.

**Comment by MSOB7YY** [comment]
u can use services like [recovermy.video](https://www.recovermy.video/) for that. It'll start saving from when u sign up and on.
also archive.org is really helpful,  just get the link of deleted video and paste it there, try different forms of the url like `www.youtube.com` `m.youtube.com` `youtu.be`

**Comment by Berny23** [comment]
> u can use services like [recovermy.video](https://www.recovermy.video/) for that. It'll start saving from when u sign up and on.
> also archive.org is really helpful,  just get the link of deleted video and paste it there, try different forms of the url like `www.youtube.com` `m.youtube.com` `youtu.be`

The maintainer of this site has mentioned that a global removed video title search would be possible based on the user data (I also already have an account).
So, maybe an API implementation similar to SponsorBlock or ReturnYouTubeDislikes in Vanced could be created if someone contacts the recovermy.video dev.
